### PR TITLE
DTSRD-435: Add Base_Location mapping for <service_code> 'BHA1'

### PR DESCRIPTION
**Pull request by fiyin-cgi**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DTSRD-435

### Change description ###
flyaway script to insert mapping as specified in ticket 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by lukasz-wolski — APPROVED** [review]

File: src/main/resources/db/migration/V1_59__alter_judicial_location_mapping_table.sql [hunk 1]
```diff
@@ -0,0 +1,4 @@
+INSERT INTO dbjudicialdata.judicial_location_mapping (epimms_id,judicial_base_location_id,building_location_name,base_location_name,service_code) VALUES
```
Comment (rajkatla-hmcts) [inline src/main/resources/db/migration/V1_59__alter_judicial_location_mapping_table.sql:1]: In ticket different schema is mentioned : dbjuddata , which one is correct ?

**Review by rajkatla-hmcts — CHANGES_REQUESTED** [review]

File: src/main/resources/db/migration/V1_59__alter_judicial_location_mapping_table.sql (hunk 1 above)
Comment (dpmatheson) [reply src/main/resources/db/migration/V1_59__alter_judicial_location_mapping_table.sql:1]: There are now 2 schemas in Judicial DB. Need input from feature team, Magnus is on the case. This change is on hold for now.

File: src/main/resources/db/migration/V1_60__insert_judicial_location_mapping_table.sql [hunk 2]
```diff
@@ -0,0 +1,4 @@
+INSERT INTO judicial_location_mapping (epimms_id,judicial_base_location_id,building_location_name,base_location_name,service_code) VALUES
```
Comment (rajkatla-hmcts) [inline src/main/resources/db/migration/V1_60__insert_judicial_location_mapping_table.sql:1]: this sql does not have schema, pls add schema.

**Review by rajkatla-hmcts — CHANGES_REQUESTED** [review]

File: src/main/resources/db/migration/V1_60__insert_judicial_location_mapping_table.sql (hunk 2 above)
Comment (dpmatheson) [reply src/main/resources/db/migration/V1_60__insert_judicial_location_mapping_table.sql:1]: the previous scripts which aren't related to elinks don't have a schema specified, so they will use the default one used in the connection string to the db which is dbjuddata

**Comment by dpmatheson** [comment]
Change to be merged on Tuesday 30/05/2023

**Review by dpmatheson — APPROVED** [review]

**Review by JackBraceyCGI — APPROVED** [review]

**Pull request merged**